### DISCUSSION TITLE
Scalability request latency

### DISCUSSION THREAD

**Discussion by snipesw** [colyseus/colyseus#779] [Q&A]
Hi all,

In my game project I use a proxy server Colyseus and Redis. At regular time intervals, it happens that sending a request on all game nodes simultaneously slows down. I have searched all the logs etc but can't find it. Does anyone have any ideas?

Thank you so much.

**Comment by endel** [comment]
Hi @snipesw, what you are experiencing is probably due to this reported issue: https://github.com/colyseus/colyseus/issues/466 

This behaviour has been reworked on version 0.16 and there's now a ["concurrent lock"](https://github.com/colyseus/colyseus/commit/c6fded628537282eab80f5c75d79132dd7eb5d36#diff-566946f7d72aa3796cbf9f855b8c287af846a2ebdfe9ccd59ae48ef1d26b6aeeR274) only for "join or create" calls.

**Reply by snipesw** [reply]
Thank you very much for your answer. I will try it.
